python/mxnet/optimizer.py

```diff
@@ -174,14 +174,24 @@ def set_lr_scale(self, args_lrscale): # pylint: disable=unused-argument
         raise DeprecationWarning
 
     def set_lr_mult(self, args_lr_mult):
-        """Set individual learning rate for each weight.
+        """Set an individual learning rate multiplier for each parameter.
+
+        If you specify a learning rate multiplier for a parameter, then
+        the learning rate for the parameter will be set as the product of
+        the global learning rate `self.lr` and its multiplier.
+
+        .. note:: The default learning rate multiplier of a `Symbol`
+            can be set with its ``__lr_mult__`` `attr`.
 
         Parameters
         ----------
         args_lr_mult : dict of string/int to float
-            Set the lr multipler for name/index to float.
-            Setting multipler by index is supported for backward compatibility,
-            but we recommend using name and symbol.
+            For each of its key-value entry,
```

Comment:
explain separately.
If you use index, it should be the same as the index you use for the weight in update method.
if you use string name, you also need to pass sym (which represent the network) in contructor